core/api/src/main/java/org/n52/sos/util/GeometryHandler.java

```diff
@@ -283,7 +283,9 @@ public void setDefaultResponse3DEpsg(final int epsgCode3D) throws ConfigurationE
     public void setSupportedCRS(final String supportedCRS) throws ConfigurationException {
         // Validation.notNull("Supported CRS codes as CSV string",
         // supportedCRS);
-        this.supportedCRS.addAll(StringHelper.splitToSet(supportedCRS, Constants.COMMA_STRING));
+        Set<String> newSupportedCRS = Sets.newHashSet();
+        newSupportedCRS.addAll(StringHelper.splitToSet(supportedCRS, Constants.COMMA_STRING));
+        this.supportedCRS = newSupportedCRS;
```

Comment:
This three lines can be combined: `this.supportedCRS = StringHelper.splitToSet(supportedCRS, ",");`